controllers/account.controller.js

```diff
@@ -36,10 +36,10 @@ async function getUserByEmail(req, res) {
  * @param {{params: {id: string}}} req
```

Comment:
should be `{{body: {id: string}}}`. Please change elsewhere.

Reply:
Done